index.html

```diff
@@ -603,29 +683,30 @@ <h1>Terminology</h1>
         shared for an entire group of <a>Things</a>. Compared to a Thing Description, a Thing Model does not contain enough 
         information to identify or interact with a Thing instance.</dd>
       <dt>
-        <dfn>Partial TD</dfn>
-      </dt>
-      <dd>
-        A <a>Partial TD</a> is an object that follows the same hierarchical structure of the <a>TD</a> information model, 
-        but it is not required to contain all the mandatory elements. 
-        <p class=note>
-          An example usage of a <a>Partial TD</a> is in <a>WoT Scripting API</a> as an input for the creation of <a>Exposed Things</a>. 
-          See the <a href="https://www.w3.org/TR/2020/NOTE-wot-scripting-api-20201124/#the-produce-method">produce</a> method for more details.
-        </p></dd>
-      <dt>
-        <dfn>TD Fragment</dfn>
+        <dfn>TD Element</dfn>
```

Comment:
We had a long discussion on these terms before we integrated them.
Please don't remove them in this MR.
If you think we need another definition, please refactor this MR.